### ISSUE TITLE
Cannot validate concrete class

### ISSUE THREAD

**Issue by shadowhand** [swaggest/php-json-schema#58]
Because of [this constraint][1] it is not possible to validate an object that has a concrete class, even if the class looks something like:

```php
class User
{
    public $id;
    public $username;
    public $email;
}
```

Shouldn't this constraint be `$ok = is_object($data);` ?

[1]: https://github.com/swaggest/php-json-schema/blob/aa5ce4073adb69f36f84255392b69b3f6588d086/src/Constraint/Type.php#L67

**Comment by vearutop** [comment]
The reason to limit objects to `\stdClass` was safety and performance of `(array)$obj` casting, though now I think it is not enough to drop class objects support. 

Thank you for pointing the issue.

**Comment by vearutop** [comment]
Fixed in [v0.12.5](https://github.com/swaggest/php-json-schema/releases/tag/v0.12.5).
